### PR TITLE
DummyTrack documentation 

### PR DESCRIPTION
**Pull request by etowahadams**
Documentation for `DummyTrack`. Corresponds to https://github.com/gosling-lang/gosling.js/pull/946

### CODE REVIEW

File: docs/composition.mdx
Comment (etowahadams) [inline docs/composition.mdx:1]: The composition page seemed like the most logical to describe `DummyTrack`. Would there be a better place?

File: docs/composition.mdx [hunk 1]
```diff
@@ -207,3 +207,33 @@ Try examples in the online editor:
...
 
 [Ideogram (text + rect + triangle)](<https://gosling-lang.github.io/gosling.js/?example=doc_overlay_rect_text>)
+
+## Placeholder track to show non-Gosling visualizations
+
+If you want to show non-Gosling visualizations within a Gosling visualization, you can use `DummyTrack` as a 
+placeholder to create space for your visualization. Your custom visualization can then be overlaid on top of 
+the `DummyTrack`. Currently, `DummyTrack` can only be used with a linear `layout`.
+
+[DummyTrack demo in the Editor](https://gosling-lang.github.io/gosling.js/?example=doc_dummy_track)
```
Comment (sehilyi) [inline docs/composition.mdx:217]: Edited a bit to provide more context.

```suggestion
## Integrating non-Gosling visualizations

You may want to visualize a non-Gosling visualization within a Gosling visualization, such as showing a lineage tree along with multiple sequences.

In Gosling, you can do this by using `DummyTrack` as a placeholder to create space for your non-Gosling visualization. Your non-Gosling visualization can then be overlaid on top of the `DummyTrack` using [JS API functions](./js-api) to get the position of the placeholder.

:::caution
`DummyTrack` can only be used with a linear `layout` at the moment.
:::

[DummyTrack demo in the Editor](https://gosling-lang.github.io/gosling.js/?example=doc_dummy_track)
```

Comment (sehilyi) [reply docs/composition.mdx:217]: We should add `gosling-meta` repository as an example later when we use DummyTrack.

**Comment by etowahadams** [comment]
Added a new page with a link in the sidebar called "Integrating non-Gosling visualizations" as described!

**Review by sehilyi — APPROVED** [review]
Thanks for the update!

**Pull request merged**